data/src/main/java/org/tuvecinoteayuda/data/regions/repository/RegionRepository.kt

```diff
@@ -4,15 +4,25 @@ import com.google.gson.Gson
 import com.google.gson.reflect.TypeToken
 import org.tuvecinoteayuda.data.DataContextWrapper
 import org.tuvecinoteayuda.data.getJsonDataFromAsset
+import org.tuvecinoteayuda.data.regions.models.City
 import org.tuvecinoteayuda.data.regions.models.Region
 
 class RegionRepository(private val contextWrapper: DataContextWrapper) {
 
+    private var regions: List<Region>? = null
```

Comment:
why we need this variable?

Reply:
to cache in memory the regions, so you don't have to parse the json again